test-case-reporting/types/test-case-reporting.d.ts

```diff
@@ -24,40 +24,59 @@ declare module 'test-case-reporting' {
 
   /** A build on Travis. */
   export interface Build {
-    username: string,
-    commitHash: string,
-    prNumber: number,
-    startedAt: Date,
+    username: string;
+    commitHash: string;
+    prNumber: number;
+    startedAt: Date;
 
     // The list of jobs we know are contained in the build.
     // When we create a build, we fill this with its jobs, but when
     // we get a build from a server, the jobs list may not be populated
     // if the jobs are not needed.
-    jobs: Array<Job>,
+    jobs: Array<Job>;
   }
 
   /** A job within a Travis build. */
   export interface Job {
-    jobNumber: string,
-    type: JobType,
-    startedAt: Date,
+    jobNumber: string;
+    type: JobType;
+    startedAt: Date;
 
     // This list is treated similarly to the `jobs` array of the `Build` type.
     // Read the comment on `jobs` for info.
-    testRuns: Array<TestRun>,
+    testRuns: Array<TestRun>;
   }
 
   /** A single kind of test case, one `it` or `test` block. */
   export interface TestCase {
-    name: string,
-    createdAt: Date,
+    name: string;
+    createdAt: Date;
   }
 
   /** An instance of a test being run, with results. */
   export interface TestRun {
-    testCase: TestCase,
-    status: TestStatus,
-    timestamp: Date,
-    durationMs: number,
+    testCase: TestCase;
+    status: TestStatus;
+    timestamp: Date;
+    durationMs: number;
+  }
+
+  namespace KarmaReporter {
+    export interface TestResultReport {
+      browsers: BrowserResultSet;
+    }
+
+    export interface BrowserResultSet {
+      results: Array<TestResult>;
+    }
+
+    export interface TestResult {
+      description: string;
+      suite: Array<string>;
+      success: boolean;
+      skipped: boolean;
+      pending: boolean;
```

Comment:
How do you plan to use `pending`?

Reply:
From what I saw, every test with `skipped: true` has `pending: true`, so I don't know exactly what `pending` means here, but it's not always false for tests that have run.

I'll remove it since we don't currently have a use for it.

Reply:
Also, I'll fix this in the db-types pull request (#907)

Reply:
I suspect `pending` is set to `false` after a test case completes, and would mainly be used to check something during the tests running. Since we are only determining this after the whole test suite is completed, it shouldn't matter to us.